apps/wakunode2/wakunode2.nim

```diff
@@ -249,6 +249,14 @@ proc initNode(conf: WakuNodeConf,
                 some(Port(uint16(conf.tcpPort) + conf.portsShift))
               else:
                 extTcpPort
+    extMultiAddrs = if (conf.extMultiAddrs.len > 0):
+                      let extMultiAddrsValidationRes = validateExtMultiAddrs(conf.extMultiAddrs)
+                      if extMultiAddrsValidationRes.isErr():
+                        return err("invalid external multiaddress: " & extMultiAddrsValidationRes.error)
```

Comment:
Here you are writing twice the "invalid external multiaddress": one in the validation function (returned as part of the error message) and another in this line.

I would suggest returning just the error in the validation and adding here the "invalid external multiaddress" message.

Reply:
Addressed in [82eff08](https://github.com/waku-org/nwaku/pull/1509/commits/82eff08658bedf1ea81474bdcf5c055f5358c349)

Removed the empty string validation too, the first check in MultiAddress.init() checks if the string starts with '/'. It covers the test case.